apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js

```diff
@@ -0,0 +1,26 @@
+import {
+  unpluggedLessonList,
+  lessonCompletedByStandard,
+  fakeStandards
+} from './standardsTestHelpers';
+
+// Action creators
+
+// Initial State
+const initialState = {};
+
+export default function sectionStandardsProgress(state = initialState, action) {
+  return state;
+}
+
+export function getUnpluggedLessonsForScript(script) {
+  return unpluggedLessonList;
+}
+
+export function getLessonsCompletedByStandardForScript(script) {
+  return lessonCompletedByStandard;
+}
+
+export function getStandardsCoveredForScript(script) {
+  return fakeStandards;
+}
```

Comment:
Wahoo! This sets us up really nicely moving forward as we continue to add functionality to this page. Thanks for adding!